assets/master.js

```diff
@@ -104,6 +105,19 @@
       byId('stats').innerHTML = '<span class="failure">' + escapeHTML(err) + '</span>';
       byId('minify-btn').disabled = false;
     });
+    byId('copy-btn').onclick = function() {
+      let bgColorButton = byId('copy-btn').style.backgroundColor;
+      let copyText = byId('output');
+      copyText.select();
+      copyText.setSelectionRange(0, 99999);
+      document.execCommand("copy");
+      byId('copy-btn').innerHTML = 'Copied!'
+      byId('copy-btn').style.backgroundColor = 'green';
+      setTimeout(function() {
+        byId('copy-btn').innerHTML = 'Copy Result';
```

Comment:
Via CSS would be probably better.

Reply:
Is this review just a duplicate of the previous one asking for CSS, or do you mean something different. I think c5d03e9 adds in using the `assets/master.css` instead of just plain editing through JS.

Reply:
It's about the same.

Reply:
You can ignore this as the change is outdated because of your changes. So it's fine now.

<img width="333" alt="Bildschirmfoto 2020-09-11 um 00 23 35" src="https://user-images.githubusercontent.com/827205/92815976-1d114780-f3c5-11ea-8de5-b01217746c6c.png">